### DISCUSSION TITLE
How to make a pixel perfect cut?

### DISCUSSION THREAD

**Discussion by pihentagy** [flameshot-org/flameshot#3486] [Q&A]
Maybe I am searching it wrong, but I cannot find a way to do the area selection in a zoomed way.

It would be fine, when I need to snip just the right size of rectangle.

**Comment by mmahmoudian** [comment]
You can enable magnifier:

![image](https://github.com/flameshot-org/flameshot/assets/390889/3ee3a8a0-f1b0-46fb-8d6f-a6dee8c43f27)


Does this address your question?

**Reply by pihentagy** [reply]
Yes, thanks, almost. But I'd prefer a zoomed in view, what shutter does: if you scroll the mouse, it zooms in to the screenshot before you select the endpoints.
